assets/src/components/notifications.tsx

```diff
@@ -64,6 +79,9 @@ export const NotificationCard = ({
       <button
         className="m-notifications__card-info"
         onClick={() => {
+          if (window && window.FS) {
+            window.FS.event("Chelsea bridge notification clicked")
```

Comment:
See previous comment on `notificationDrawer.tsx`.

Reply:
Guess which part of the code I wrote directly before leaving for the day on Friday.